src/lint/lint.test.ts

```diff
@@ -24,27 +24,28 @@ before(() => {
 });
 
 describe("#printResults", () => {
-  let consoleSpy: Sinon.SinonSpy;
+  let consoleStub: Sinon.SinonSpy;
 
   beforeEach(() => {
-    consoleSpy = sinon.spy(console, "log");
+    sinon.restore();
```

Comment:
Is this added to override/restore the stubbing done in setup.ts? Just trying to understand in the future if we want to stub/spy the logger in a specific test, we have to first restore it. Is that correct?

Reply:
correct